xtask/codegen/css.ungram

```diff
@@ -953,8 +962,31 @@ CssParameterList = CssParameter (',' CssParameter)* ','?
 //			    ^^^
 // repeating-radial-gradient(red, yellow 10%, green 15%);
 //                                ^^^^^^^^^^
-CssParameter = CssComponentValueList
+CssParameter = AnyCssParameter
+AnyCssParameter = AnyCssExpression | CssListOfComponentValuesExpression
+
+AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression |  AnyCssValue
+
+CssListOfComponentValuesExpression = CssComponentValueList
```

Comment:
Rather than making an intermediate `CssListOfComponentValuesExpression`, could we just reuse `CssComponentValueList` directly in `AnyCssComponentValue`? I'm not sure the intermediate node provides any useful additional information, and can make analysis and formatting a little more confusing to deal with (like checking if a value list is in a parameter can no longer just check `node.parent() == AnyCssParameter`, it has to jump through the intermediary.

If there's a possibility that the expression list could have a different grammar, then I think it makes sense to keep. Otherwise, I think it should be removed and just use `CssComponentValueList` directly.

Reply:
> Rather than making an intermediate `CssListOfComponentValuesExpression`, could we just reuse `CssComponentValueList` directly in `AnyCssComponentValue`? I'm not sure the intermediate node provides any useful additional information, and can make analysis and formatting a little more confusing to deal with (like checking if a value list is in a parameter can no longer just check `node.parent() == AnyCssParameter`, it has to jump through the intermediary.
> 
> If there's a possibility that the expression list could have a different grammar, then I think it makes sense to keep. Otherwise, I think it should be removed and just use `CssComponentValueList` directly.

The reason I added the `CssListOfComponentValuesExpression` is because if I just use the
```
CssParameter = AnyCssExpression | CssComponentValueList

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression | AnyCssValue
```

I have also tried the following
```
CssParameter = AnyCssExpression

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression | CssComponentValueList | AnyCssValue

```
```
CssParameter = AnyCssExpression

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression | AnyCssComponentValue

AnyCssComponentValue =  CssComponentValueList | AnyCssValue
```
```
CssParameter = AnyCssExpression

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression | CssComponentValueList

```
```
CssParameter = AnyCssExpression

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression | AnyCssComponentValue

AnyCssComponentValue =  CssComponentValueList 
```
```
CssParameter = AnyCssExpression 

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression |  AnyCssValue | CssListOfComponentValuesExpression

CssListOfComponentValuesExpression = CssComponentValueList
```
```
AnyCssParameter = AnyCssExpression  

AnyCssExpression =  CssBinaryExpression | CssParenthesizedExpression | CssComponentValueList
```

I get the following error

```
error[E0560]: struct `nodes::CssComponentValueList` has no field named `syntax`                                                                                                                                                  
     --> crates\biome_css_syntax\src\generated\nodes.rs:10606:81
      |
10606 |                 AnyCssExpression::CssComponentValueList(CssComponentValueList { syntax })
      |                                                                                 ^^^^^^ `nodes::CssComponentValueList` does not have this field
      |
      = note: available fields are: `syntax_list`

error[E0609]: no field `syntax` on type `&nodes::CssComponentValueList`
     --> crates\biome_css_syntax\src\generated\nodes.rs:10623:64
      |
10623 |             AnyCssExpression::CssComponentValueList(it) => &it.syntax,
      |                                                                ^^^^^^ unknown field
      |
      = note: available fields are: `syntax_list`

error[E0609]: no field `syntax` on type `nodes::CssComponentValueList`
     --> crates\biome_css_syntax\src\generated\nodes.rs:10631:63
      |
10631 |             AnyCssExpression::CssComponentValueList(it) => it.syntax,
      |                                                               ^^^^^^ unknown field
      |
      = note: available fields are: `syntax_list`

error[E0277]: the trait bound `biome_rowan::SyntaxNode<syntax_node::CssLanguage>: From<nodes::CssComponentValueList>` is not satisfied                                                                                           
     --> crates\biome_css_syntax\src\generated\nodes.rs:10652:63
      |
10652 |             AnyCssExpression::CssComponentValueList(it) => it.into(),
      |                                                               ^^^^ the trait `From<nodes::CssComponentValueList>` is not implemented for `biome_rowan::SyntaxNode<syntax_node::CssLanguage>`
      |
```
Do you have any good advice?

Reply:
I changed the grammer and now it looks like this, do you think it's ok?
```
CssParameterList = CssParameter (',' CssParameter)* ','?

// cubic-bezier(0.1, 0.7, 1.0, 0.1)
//			    ^^^
// repeating-radial-gradient(red, yellow 10%, green 15%);
//                                ^^^^^^^^^^
CssParameter = AnyCssExpression 

AnyCssExpression = CssBinaryExpression | CssParenthesizedExpression |  CssListOfComponentValuesExpression 
CssListOfComponentValuesExpression = CssComponentValueList

CssBinaryExpression = 
	left: AnyCssExpression
	operator_token: (
		'+' | '-' | '*' | '/' 
	)
	right: AnyCssExpression


CssParenthesizedExpression =
	'(' expression: AnyCssExpression? ')'

```

Reply:
I think this looks good! I wasn't aware of the limitation for not using List nodes in an Any node, but it makes sense. Given that, it makes sense to break it out this way